### PR TITLE
Refrigerator screen and item model

### PR DESCRIPTION
**Pull request by hamzzi**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: close #

## What is the new behavior?
- Refrigerator screen
  - add `ListView.builder` for items
  - add `floatingActionButton` to add items
  - add `onTap` to delete items
- item model for refrigerator

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: best_before/lib/models/item.dart [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class Item {
+  String name;
+  double quantity;
+  String measurement;
```
Comment (swoohooh) [inline best_before/lib/models/item.dart:4]: 이부분은 enum class 로 대체하는게 좋겠네요.

File: best_before/lib/models/item.dart [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class Item {
```
Comment (swoohooh) [inline best_before/lib/models/item.dart:1]: 너무 범용적인 이름이라 나중에 수정이 필요할 것 같은데, 음... 뭐라고 바꾸면 좋을지는 모르겠네요

File: best_before/lib/screens/refrigerator_screen.dart [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    Duration diff = toDay.difference(expirationDay);
+
+    if (diff.inDays < 0) {
+      return '${(diff.inDays).abs()} days remain';
+    } else {
+      return '${(diff.inDays).abs()} days past';
+    }
+  }
+
+  void _creatItem() {
```
Comment (swoohooh) [inline best_before/lib/screens/refrigerator_screen.dart:62]:
```suggestion
  void _createItem() {
```

File: best_before/lib/screens/refrigerator_screen.dart [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+                onTap: () => setState(() {
+                  items.removeAt(index);
+                }),
+                title: Text(items[index].name),
+                subtitle: Text(
+                    '${items[index].quantity} ${items[index].measurement}'),
+                trailing: Text(checkExpiration(items[index])),
+              ))),
+      floatingActionButton: FloatingActionButton(
+        onPressed: _creatItem,
```
Comment (swoohooh) [inline best_before/lib/screens/refrigerator_screen.dart:95]:
```suggestion
        onPressed: _createItem,
```

**Review by swoohooh — COMMENTED** [review]
:+1: 
issue 에 폴더 구조에 대해서 고민해보고 올려놓겠습니다.

File: best_before/lib/screens/refrigerator_screen.dart (hunk 3 above)
Comment (hamzzi) [reply best_before/lib/screens/refrigerator_screen.dart]: 수정했습니다.

File: best_before/lib/models/item.dart (hunk 1 above)
Comment (hamzzi) [reply best_before/lib/models/item.dart]: 수정했습니다.

**Review by swoohooh — DISMISSED** [review]
굳

**Review by swoohooh — DISMISSED** [review]

**Comment by swoohooh** [comment]
> PR 반영 내역 때문에 forch-push 했습니다 😢

4... 시간 전???????????????????????

**Review by swoohooh — APPROVED** [review]

**Pull request merged**